modules/core/src/main/java/org/apache/synapse/api/API.java

```diff
@@ -477,6 +496,15 @@ void process(MessageContext synCtx) {
         }
     }
 
+    private boolean isBound(Resource resource, MessageContext synCtx) {
+        Collection<String> bindings = resource.getInboundEndpointBindings();
```

Comment:
please check whether de serialization shows these attributes in ui

Reply:
Checked and confirmed that these attributes were not shown, resolved it with 6512d31.